pyblazing/pyblazing/apiv2/context.py

```diff
@@ -3053,6 +3094,27 @@ def sql(
                         query_table.input = [query_table.input]
                     currentTableNodes.append(query_table)
 
+            elif (
```

Comment:
what is actually happening here? this is just copied from when you have a file. How does the engine distribute pulling data from mySQL from different nodes?

Reply:
This is the only remain issue, we keep working on this one cc @gcca

Reply:
ok I just sent a push, that should fix this issue ... tomorrow morning we are going to run some test to make sure distribution is fine, good catch @williamBlazing thanks!

Reply:
Distribution is working fine now, thanks!